python/cuml/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -639,19 +618,17 @@ def test_standardization_on_normal_dataset(
     C = regularization[1]
     l1_ratio = regularization[2]
 
-    nrows = int(1e5) if n_classes < 5 else int(2e5)
-
     # test correctness compared with scikit-learn
     lr = _test_lbfgs(
-        nrows=nrows,
+        nrows=int(1e5),
         ncols=20,
         n_parts=2,
         fit_intercept=fit_intercept,
         datatype=datatype,
         delayed=delayed,
         client=client,
         penalty=penalty,
-        n_classes=n_classes,
+        n_classes=2,
```

Comment:
For cases like this where we now only test one case of `n_classes` should we pick the "unusual" value of `3` instead of `2`? I'm thinking that probably most people check the two class case, but for more than two classes things get less well tested -> make the automated test use 3

Reply:
My thought was we already have plenty of tests in this file for multiclass, so for tests checking a specific non-class-related feature defaulting to the simpler `n_classes=2` made more sense.

Reply:
Good point, I had only thought about the diff, not the rest of the file